sdk/core/azure-core/src/http/policy.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <http/http.hpp>
+#include <http/policy.hpp>
+
+using namespace Azure::Core::Http;
+
+std::unique_ptr<Response> NextHttpPolicy::ProcessNext(Context& ctx, Request& req)
+{
+  if (m_policies == nullptr)
+    throw;
```

Comment:
Can we at least throw some useful string message to help debug any situation we get into here?

Reply:
Yep, the docs and errors will get better once the architecture is agreed on.